help_data.yaml

```
@@ -15,10 +15,10 @@ cis:
 
 esm-apps:
     help: |
-      UA Apps: Extended Security Maintenance is enabled by default on entitled
-      workloads. It provides access to a private PPA which includes available
-      high and critical CVE fixes for Ubuntu LTS packages in the Ubuntu Main
-      and Ubuntu Universe repositories from the Ubuntu LTS release date until
+      Ubuntu Pro: Extended Security Maintenance (ESM) Apps is enabled by default
```

This is a combination of the long and short names. I think we should just use the long name
```suggestion
      Extended Security Maintenance for Applications is enabled by default
```